pkg/core/config/writer.go

```diff
@@ -110,6 +110,20 @@ type WriterConfig struct {
 	// handle the volume of trace spans and should be upgraded to more powerful
 	// hardware/networking.
 	MaxTraceSpansInFlight uint `yaml:"maxTraceSpansInFlight" default:"100000"`
+	// Enable logging to a Splunk Enterprise instance
+	LogSplunkEnabled bool `yaml:"splunkEnabled"`
```

Comment:
The fact that all of these config values start with `splunk` makes me think they should be put into a common config struct nested within this struct.